blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+
+As part of the Serverless Workflow implementation, Kogito offers a Kubernetes Operator to deploy these workflows with Knative. The goal is to make it as simple as possible to deploy and manage user-defined workflows in  cloud environments. Knative Eventing plays a very important role in this scenario by providing the underlying infrastructure for event-driven architectures.
+
+### Kogito Serverless Workflow
+
+To demonstrate how the Kogito workflow implementation works on event-driven architectures, we will use the [Patient Onboarding example](https://github.com/kiegroup/kogito-examples/tree/master/serverless-workflow-functions-events-quarkus). In this example, we simulate a workflow used in hospitals to onboard new patients and assign them to the correct doctor.
+
+The following image taken from the [specification examples page](https://github.com/serverlessworkflow/specification/blob/master/examples/examples.md#New-Patient-Onboarding) illustrates this workflow:
+
+![Patient Onboarding workflow representation](https://raw.githubusercontent.com/serverlessworkflow/specification/master/media/examples/example-patientonboarding.png)*Patient Onboarding workflow representation*
+
+The workflow will start after receiving a [Cloud Event](https://github.com/cloudevents/spec) with patient information, and then will call three functions in a sequence which: (1) store the patient information, (2) assign to a doctor based on their symptoms, and (3) schedule an appointment with the assigned doctor.
+
+This is how the Patient Onboarding workflow looks like based on the specification:
+
+```yaml
+id: onboarding
+version: '1.0'
+name: Patient Onboarding Workflow
+states:
+- name: Onboard
+  type: event
+  start:
+    kind: default
+  onEvents:
+  - eventRefs:
+    - NewPatientEvent
+    actionMode: sequential
+    actions:
+    - functionRef:
+        refName: StoreNewPatient
+    - functionRef:
+        refName: AssignPatientToDoctor
+    - functionRef:
+        refName: SchedulePatientAppointment
+  end:
+    kind: default
+events:
+- name: NewPatientEvent
+  type: new.patient.events
+  source: "/hospital/entry"
+functions:
+- name: StoreNewPatient
+  operation: classpath:openapi.json#storeNewPatient
+- name: AssignPatientToDoctor
+  resource: classpath:openapi.json#assignPatientToDoctor
+- name: SchedulePatientAppointment
+  resource: classpath:openapi.json#schedulePatientAppointment
+```
+
+The above workflow definition is based on three parts. The first part is the “states” array which includes the workflow control-flow logic, in our case the “event” state which receives the new patient event and performs our three actions (service invocations).
```

Comment:
```suggestion
This specification is based on three parts. The first part is the `states` array, which includes the workflow control flow logic. In this case, the state is an `event`, which receives the new patient cloud event, and performs the three functions mentioned previously.
```

Reply:
Hi @abrennan89 many thanks for your review! :smile: 

This paragraph is about the workflow definition listed above. I think "This specification" won't reflect that, so the reader wouldn't associate the workflow and the description that follows it.

Reply:
Would it be more clear to start that sentence with
"The above workflow definition contains three mayor parts ...."
The entire workflow definition complies with the specification. Just trying to figure out what you think would be best wording for it.

Reply:
That's fine - could you please make the changes to `states` etc though, since they're literal fields so it seems like they should be formatted like `this` instead of "this"? I don't mind what way you choose to word this part though, and thanks for clarifying :)